### PR TITLE
Add `Domain.listDomains` and `Domain.getProperties` APIs

### PR DESCRIPTION
**Pull request by labkey-kevink**
- add `Domain.listDomains` (LabKey/platform@252d8c2fd3c7c329d1d4cdaeaaaf97ccbb295672)
- add `Domain.getProperties` (LabKey/platform@cd4287c656954d4d4ef9d65b9dc3fe2595cac260)
- add `Exp.loadRuns`

### CODE REVIEW

File: src/labkey/Exp.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class ExpObject {
     /**
      * The person who created the ExpObject.
      */
-    createdBy: number;
+    createdBy: string;
```
Comment (labkey-martyp) [inline src/labkey/Exp.ts]: These are not LK user ids?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/api",
-  "version": "0.0.25",
+  "version": "0.0.27-fb-provenance-runbuilder.0",
```
Comment (labkey-martyp) [inline package.json]: Don't forget to change version

File: src/labkey/Domain.ts [hunk 3]
```diff
@@ -193,4 +193,64 @@ export function save(config: SaveDomainOptions): void {
...
+
+interface GetPropertiesParams {
+    propertyIds?: Array<number>
+    propertyURIs?: Array<string>
+}
+
+export interface GetPropertiesOptions extends GetPropertiesParams{
+    containerPath?: string
+    failure?: (error?: any) => any
+    success?: (response: {data?: Array<any /*DomainField*/>}) => any
```
Comment (labkey-martyp) [inline src/labkey/Domain.ts]: Is this supposed to be DomainField?

File: src/labkey/Exp.ts (hunk 1 above)
Comment (labkey-kevink) [reply src/labkey/Exp.ts]: They are user email addresses serialized in [ExperimentJSONConverter.serializeStandardProperties](https://github.com/LabKey/platform/blob/10719867f28056fc5e26a78467e33af992609927/api/src/org/labkey/api/exp/api/ExperimentJSONConverter.java#L154)

File: src/labkey/Domain.ts (hunk 3 above)
Comment (labkey-kevink) [reply src/labkey/Domain.ts]: Yes, it is a DomainField, but labkey-api-js doesn't (yet) have the DomainField type defined in our components package.

File: src/labkey/Exp.ts (hunk 1 above)
Comment (labkey-tchad) [reply src/labkey/Exp.ts]: Will users that aren't able to view emails not see `createdBy` information?

**Pull request merged**